app/pages/setupquestions/lifetimeallowance/KickOutStatusPage.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.setupquestions.lifetimeallowance
+
+import models.KickOutStatus
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object KickOutStatusPage extends QuestionPage[KickOutStatus] {
```

Comment:
should this be a "page"?

Reply:
or should this just all be part of it's own model?

Reply:
moved to model